package.json

```diff
@@ -41,7 +41,7 @@
   "homepage": "https://github.com/asyncapi/parser-js",
   "devDependencies": {
     "@semantic-release/commit-analyzer": "^8.0.1",
-    "@semantic-release/github": "^7.0.4",
+    "@semantic-release/github": "7.2.3",
     "@semantic-release/npm": "^7.0.3",
     "@semantic-release/release-notes-generator": "^9.0.1",
```

Comment:
You can update also this package to `9.0.2`, but it's optional.

Reply:
yeah I don't want to 😄